src/Nri/Ui/DisclosureIndicator/V2.elm

```diff
@@ -16,37 +16,22 @@ import Nri.Ui.Svg.V1 as NriSvg
 
 type alias Config =
     { isOpen : Bool
+    , size : Css.Px
```

Comment:
Are there standard sizes from design that we should be encouraging here? Have we needed PX values other than the previous medium and smalls?

Reply:
Ooo great question.

@NoRedInk/design ? is there a single size that the disclosure caret should be? It looks like the zeplin has the Expander and List section carets as always being 15px -- is that what you want?

Reply:
I think having a few different sizes would be good. We could go with 15px and 17px for now

Reply:
Thanks, Dansby!

@brookeangel would you like me to change the API or the examples at all?

Reply:
If we think we won't be changing the sizes, I think maybe the API to encourage people to use the correct rather than guessing? What do you think?

Reply:
Yeah that sounds good! changing it :)

Reply:
How does it look now?